### PR TITLE
Snapscan - Budget App

### PR DESCRIPTION
**Pull request by julzedz**
Hi reviewer,
In this project, I was able to do the following:

- Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the signup and log-in pages.

- Sign-up and login pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

- Home page (categories page)

When the user logs in, they are presented with the categories page.

For each category, the user can see their name, icon, and the total amount of all the transactions that belong to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

- Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user clicks (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

- Testing requirements

Create unit and integration tests for all the most important components of your RoR application.
Also, deploy the app using render and add a link to the video presentation as well in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- **Transactions page**
+- **Add a new category page**
+- **Add a new transaction page**
+- **Use devise for user authentication**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- <a href="https://snapscan-89o4.onrender.com/">Live Demo</a>
+- [Video Presentation](https://www.loom.com/share/a5387e7ca63342988e5defaa5607dd94?sid=55bafea5-ea2f-4bec-ac24-3ee12e540bb7)
+
```
Comment (Lizdev-05) [inline README.md:72]: - [ ] To meet the project requirements, I suggest that you include a highlight in your video presentation that showcases a piece of code you find interesting or are proud of creating. Therefore, ensure that you display some code in the presentation and explain what makes you feel proud. :pray:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+# 📖 Snapscan <a name="about-project"></a>
+
+
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+<div align="center">
+  <img src="./app/assets/images/diagram.png" alt="logo" />
+  <br/>
+</div>
```
Comment (Lizdev-05) [inline README.md]: - [ ] Please this link is a broken link in your readme file so make sure that you remove or fix this link to make your README file professional :pray:

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @julzedz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 
:heavy_check_mark: All your tests pass :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (julzedz) [reply README.md:72]: Hello @Lizdev-05, I mentioned in the video about the piece of code(method) I added to the group model to limit it to just 5 transactions. I also mentioned the transitions I added to the css styling. Both of these are not part of the requirements but I added and mentioned them in the video, that I'm proud of those.
Kindly attend to this as soon as you can, thank you.

**Review by andersonlebon — APPROVED** [review]
## Hi @julzedz 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

nodding-yes

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**